spinoso-time/src/time/chrono/build.rs

```diff
@@ -50,6 +51,38 @@ impl Time {
             offset,
         }
     }
+
+    /// Creates a new `Time` object from the `seconds` and `sub_seconds_nanos`
+    /// since the Unix EPOCH with a local offset.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use spinoso_time::Time;
+    /// let epoch = Time::at(0, 0);
+    /// let epoch_plus_1_nano = Time::at(0, 1);
+    /// ```
+    #[inline]
+    #[must_use]
+    pub fn at(seconds: i64, sub_second_nanos: i64) -> Self {
```

Comment:
oh I see, I like this better than my suggestion above to keep the `i64` type on `sub_second_nanos`, but if we do that, the conversion is fallible, so I'd like to see us return `Option<Self>`. This way, when we use checked addition and checked conversions to `u32`, we can fail.

In this case, I'd prefer to return `Option` rather than `Result` since the only way constructing a `Time` with `at` can fail is if the failure is of the `Time is too large` case, which we can document.